lib/game_logic.rb

```diff
@@ -0,0 +1,101 @@
+require('rainbow')
+# rubocop:disable Style/GlobalVars,Metrics/MethodLength
+$board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+$winner = false
```

Comment:
**Important** please avoid the use of global variables. That is not a good practice. Read [this article](https://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/globalvars.html) to know why. Besides, there is no reason to use it. You might as well make it an instance variable and make it accessible to other classes if you need to. Right now, you should move them inside a class (`$board` to Board and `$winner` to Game)